app/models/issue.rb

```diff
@@ -39,7 +39,8 @@ class Issue < ApplicationRecord
   scope :ordered, -> { order(created_at: :desc) }
   scope :approved, -> { where(status: :opened) }
   scope :closed, -> { where(status: :closed) }
-  scope :find_issues, ->(a) { where('title like ?', "%#{a}%") }
+  search_query = 'title like :args OR users.last_name like :args'
```

Comment:
I don't need it. But in one line scope become too long for rubocop, and if divide it in two lines, rubocop demands to use lambda, and code become even more complicated than with this variable...